src/gui/transferlistwidget.cpp

```diff
@@ -257,7 +257,7 @@ void TransferListWidget::setSelectedTorrentsLocation()
 
     const QString newLocation = QFileDialog::getExistingDirectory(this, tr("Choose save path"), oldLocation,
                                             QFileDialog::DontConfirmOverwrite | QFileDialog::ShowDirsOnly | QFileDialog::HideNameFilterDetails);
-    if (!QDir(newLocation).exists()) return;
+    if (newLocation.isEmpty() || !QDir(newLocation).exists()) return;
```

Comment:
I wonder why the original check didn't work? Qt bug? Or does QDir always think that an empty path exists?